src/app/app.module.ts

```diff
@@ -161,15 +163,17 @@ import { ClientEffects } from './components/shared-components/store/effects/clie
     MatSnackBarModule,
     ClipboardModule,
     StoreModule.forRoot({
+      app: loginReducer,
```

Comment:
Refactor: app: loginReducer, this line is not being used, the line token: loginReducer is being used. These two are doing the same thing

Reply:
i see